docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -79,24 +79,25 @@ Alluxio UFS root mount point in the Alluxio docker image:
   ```console
   $ docker run -v /alluxio_ufs:/opt/alluxio/underFSStorage   ...
   ```
-  Of course, you can choose different path `/alluxio_ufs` you like to mount but please make sure it
-  is writable.
+Of course, you can choose different path `/alluxio_ufs` you like to mount but please make sure it
```

Comment:
nit: not related to this PR but the grammar seems to be wrong. Change this to "..., you can choose to mount a different path instead of `/alluxio_ufs`. Please make sure it is writable by the user the Docker image is run as.